pkg/servicescm/servicescm.go

```diff
@@ -47,11 +47,12 @@ type NodeCmdArg struct {
 	NodeObjectJsonPath string `json:"nodeObjectJsonPath"`
 }
 
-// PowershellCmdArg describes a PowerShell variable and how its value can be populated
-type PowershellCmdArg struct {
-	// Name is the variable name as it appears in commands
-	Name string `json:"name"`
-	// Path is the location of a PowerShell script whose output is the value of the variable
+// PowershellPreScript describes a PowerShell script to be ran and an optional variable to be populated
+type PowershellPreScript struct {
+	// VariableName is the name of a variable which should be replaced by the output of the script. An empty value will
+	// cause no variable replacement to occur, but the script will still be ran.
+	VariableName string `json:"variableName"`
```

Comment:
Do we need to add `omitempty`?

Reply:
Yes, I think so. In addition, would be nice to add a unit test to cover this scenario, where a `PowershellPreScript`  gets created without `VariableName`.